src/assets/translations/fr/main.json

```diff
@@ -231,10 +246,11 @@
     "notSupported": "%{asset} n'est pas pris en charge par %{wallet}"
   },
   "connectWalletPage": {
+    "title": "L'échange multichaîne original",
     "shapeshift": "ShapeShift",
     "exploreThe": "Explorez",
     "defiUniverse": "l'Univers DeFi",
-    "body": "100% Gratuit. Open‑Source. Propriété communautaire. Non‑dépositaire. Privé.",
+    "body": "Échangez, déplacez et gagnez. Privé. Propriété communautaire. Non‑dépositaire. Décentralisé.",
```

Comment:
Not sure about the `bridge` translation here. Maybe let's keep it more English for this one instance i.e "bridgez"?

Reply:
You're right it misses some of semantic. As discussed in Discord, we've settled to use a neologism/frenglish "Bridger" for now and "Transfer/Transfert" when context is present. I've updated the translations across the app based on this. Thanks for the review !